lib/relation-definition.js

```diff
@@ -3008,8 +3010,10 @@ EmbedsMany.prototype.build = function(targetModelData) {
 
   if (this.definition.options.prepend) {
     embeddedList.unshift(inst);
+    modelInstance[propertyName] = embeddedList;
   } else {
     embeddedList.push(inst);
+    modelInstance[propertyName] = embeddedList;
```

Comment:
Sync parent instance's embedded property when `embeddedList` refers to a dangle `[]`